src/library.rs

```diff
@@ -6,6 +6,34 @@ use nameutil::split_namespace_name;
 use traits::*;
 use version::Version;
 
+use stripper_interface as stripper;
+use stripper_interface::Type as SType;
+use stripper_interface::TypeStruct;
+
+pub trait ToStripperType {
```

Comment:
This trait and its implementation for lack of better place should go to the `codegen::doc` module.